executor/distsql.go

```diff
@@ -589,6 +593,7 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, workCh chan<
 			SetIsStaleness(e.isStaleness).
 			SetFromSessionVars(e.ctx.GetSessionVars()).
 			SetFromInfoSchema(e.ctx.GetInfoSchema()).
+			SetClosestReplicaReadAdjuster(newClosestReadAdjuster(e.ctx, &builder.Request, e.idxNetDataSize/float64(len(kvRanges)))).
```

Comment:
why `/float64(len(kvRanges))`?

Reply:
Because the `idxNetDataSize ` is the estimated total size of all partitions, here we need to use the cost per-partition.

Reply:
sorry, I mean regions.